wallet/src/logger.rs

```diff
@@ -6,6 +6,7 @@ use log::{Level, Log, Metadata, Record, SetLoggerError};
 
 struct Logger {
     level: Level,
+    stream: io::Stderr,
 }
 
 impl Log for Logger {
```

Comment:
we could use this to print the long in the terminal UI, I think this will be a good feature to have, but not sure that buffering the log will be a good solution, maybe some good research feature?

Reply:
Yeah, agreed it could be nice to have a logs tab. Right now what I do is I just `2> wallet.log` and open a `tail -f wallet.log` in a different window.